docs/runbooks/SSPHighRateRejectedVms.md

```diff
@@ -0,0 +1,24 @@
+# SSPHighRateRejectedVms
+
+## Meaning
+
+This alert indicates that many Virtual Machines were not valid and therefore were not created or not modified.
+This means either a user or a script is continuously trying to create invalid Virtual Machines.
```

Comment:
@borod108 "It" means instead of "This" means.

Reply:
done